client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -129,6 +133,7 @@ export default function SitesTableRow( { site }: SiteTableRowProps ) {
 	} else if ( site.is_private ) {
 		siteStatusLabel = __( 'Private' );
 	}
+	const setmShotsUrl = ! isComingSoon && ! site.is_private;
```

Comment:
How about using a more descriptive name here?

```suggestion
	const shouldUseScreenshot = ! isComingSoon && ! site.is_private;
```

Reply:
yeah that make sense